docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -540,64 +547,67 @@ spec:
   - pre-adoption-validation
 EOF
 ----
-+
-Wait for the validation to finish.
+
+.. Wait for the validation to finish.
 
 .. Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
```

Comment:
is this line meant to be a hidden comment?

Reply:
Yeah, that looks like a comment that doesn't need to be there, and the two commands in that step could probably  be combined.
$ watch oc get pod -l app=openstackansibleee
$ oc logs -l app=openstackansibleee -f --max-log-requests 20